app/mailers/user_mailer.rb

```diff
@@ -1,11 +1,18 @@
 class UserMailer < ActionMailer::Base
   default from: Rails.configuration.branding[:organisation][:email]
 
+  @app_name = Rails.configuration.branding[:application][:name]

```

Comment:
nice refactoring here, however I'm not fan of initialising instance variables outside of its constructor. The two constants only apply for feedback notification method and are gettext functions indeed, I would not create constants for that